src/test/java/com/ericsson/ei/handlers/test/RmqPropertiesTest.java

```diff
@@ -0,0 +1,103 @@
+/*
+   Copyright 2017 Ericsson AB.
+   For a full list of individual contributors, please see the commit history.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+package com.ericsson.ei.handlers.test;
+
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertThat;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.TestPropertySource;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+
+import com.ericsson.ei.handlers.RMQProperties;
+
+
+@ContextConfiguration(classes = RMQProperties.class)
+@TestPropertySource(locations = "classpath:RmqHandlerTestValues.properties")
+@RunWith(SpringJUnit4ClassRunner.class)
+
+public class RmqPropertiesTest {
+
+    private Boolean queueDurable = true;
```

Comment:
We usually set constants like these as 'private static final' with names in upper case.